crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -597,6 +750,24 @@ impl MeshPipeline {
             ))
         }
     }
+
+    pub fn get_view_layout(&self, layout_key: MeshPipelineViewLayoutKey) -> &BindGroupLayout {
+        let index = layout_key.bits() as usize;
+        let layout = &self.view_layouts[index];
+
+        #[cfg(debug_assertions)]
```

Comment:
iirc @mockersf have been raising concerns wrt using `cfg(debug_assertions)`.

Reply:
Hmm, I couldn't find it on Slack, is there a writeup on that? Is it because of the conflation between "debug builds", "builds with debug symbols" and "builds without assertions?" Do we have a feature flag or some other static way to disable this check if needed?

Reply:
I might have taken crazy pills here. I can't find it.